test_sklearn_questions.py

```diff
@@ -35,15 +37,23 @@ def test_one_nearest_neighbor_check_estimator(k):
     check_estimator(KNearestNeighbors(n_neighbors=k))
 
 
-def test_time_split():
+@pytest.mark.parametrize("end_date_and_splits",
+                         [('2021-01-31', 12), ('2020-12-31', 11)])
+@pytest.mark.parametrize("shuffle_data", [True, False])
+def test_time_split(end_date_and_splits, shuffle_data):
 
-    date = pd.date_range(start='2020-01-01', end='2021-01-31', freq='M')
+    date = pd.date_range(start='2020-01-01',
+                         end=end_date_and_splits[0], freq='M')
```

Comment:
Make it a bit harder, to make sure we have multiple sample per split.
```suggestion
                         end=end_date_and_splits[0], freq='D')
```